.github/workflows/sync-branches.yml

```diff
@@ -74,9 +74,11 @@ jobs:
         run: |
           git config user.name "github-actions[bot]"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-
+          # Activate the ours merge driver to respect any branch specific files
+          git config merge.ours.driver true
           git merge "$base_branch" --strategy=ort --strategy-option=theirs
 
+          # Must be kept in sync with .gitattributes
```

Comment:
See the PR description for details about consolidating this duplicate list -- tl;dr is that we could likely do it but it should be a separate effort.